src/twisted/internet/gtk2reactor.py

```diff
@@ -89,19 +79,7 @@ def __init__(self, useGtk=True):
         _glibbase.GlibReactorBase.__init__(self, gobject, _gtk, useGtk=useGtk)
 
 
-class PortableGtkReactor(_glibbase.PortableGlibReactorBase):
-    """
-    Reactor that works on Windows.
-
-    Sockets aren't supported by GTK+'s input_add on Win32.
-    """
-
-    def __init__(self, useGtk=True):
-        _gtk = None
-        if useGtk is True:
-            import gtk as _gtk
-
-        _glibbase.PortableGlibReactorBase.__init__(self, gobject, _gtk, useGtk=useGtk)
+PortableGtkReactor = Gtk2Reactor
```

Comment:
Just a quick comment.

Not doing a full review as I see the test are failing.

I guess that under normal circumstances, we would raise an exception when this is used/imported.

I agree that it is not worth the trouble.

But it would help to have an inline comment as a shortcut for `git blame`.

```suggestion
# We don't bother deprecating the PortableGtkReactor.
# The original code was removed and replaced with the
# backward compatible generic GTK reactor.
PortableGtkReactor = Gtk2Reactor
```